react-native-libraries.json

```
@@ -12935,5 +12935,12 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/fnando/i18n",
+    "ios": true,
```

Hello @alexisgahon, thanks for adding new entry to the directory! 👍 

Looks like this library name differs from the repository name, in those cases we can specify "npmPkg" manually:

```suggestion
    "githubUrl": "https://github.com/fnando/i18n",
    "npmPkg": "i18n-js",
    "ios": true,
```